frame/contracts/src/lib.rs

```diff
@@ -186,12 +186,7 @@ pub mod pallet {
 	use sp_runtime::Perbill;
 
 	/// The current storage version.
-	#[cfg(not(any(test, feature = "runtime-benchmarks")))]
-	const STORAGE_VERSION: StorageVersion = StorageVersion::new(13);
-
-	/// Hard coded storage version for running tests that depend on the current storage version.
-	#[cfg(any(test, feature = "runtime-benchmarks"))]
-	const STORAGE_VERSION: StorageVersion = StorageVersion::new(2);
+	pub(crate) const STORAGE_VERSION: StorageVersion = StorageVersion::new(13);
```

Comment:
I kind of like reading the version directly here rather than pulling it from the codgen.
Alternatively we just use `codegen::LATEST_MIGRATION_VERSION` (and get rid of the test)

Reply:
Yes I agree. Keeping it explicit is better.